train.py

```diff
@@ -208,10 +208,11 @@ def train(tasks, model, args, device):
         model.to(device)
         print("Tasks")
         tasks = []
-        for emotion in SemEval18SingleEmotionTask.EMOTIONS:
-            tasks.append(SemEval18SingleEmotionTask(emotion))
-        tasks.append(SarcasmDetection())
-        tasks.append(OffensevalTask())
+        # for emotion in SemEval18SingleEmotionTask.EMOTIONS:
+        #     tasks.append(SemEval18SingleEmotionTask(emotion))
+        # tasks.append(SarcasmDetection())
+        # tasks.append(OffensevalTask())
```

Comment:
Please, uncommet this

Reply:
Sorry about that! I uncommented it in a new commit below